1.1.0 breaks require() usage

Looks like the build process changes between `1.0.0` and `1.1.0` broke the ability to use this as a Common JS module. Here's what I get post upgrade:

```
TypeError: SimpleTable is not a constructor
```
or sometimes
```
Cannot find module 'cli-simple-table'
```
or from eslint
```
import/no-unresolved: Unable to resolve path to module 'cli-simple-table'.
```

Versions:
NPM: 6.14.13
Node: v14.17.3

--- a/src/index.ts
+++ b/src/index.ts
@@ -135,4 +135,4 @@ class SimpleTable {
 	}
 }
 
-export default SimpleTable;
+export = SimpleTable;

